### ISSUE TITLE
[sensor] Please add sensor.timestamp following W3C sensor

### ISSUE THREAD

**Issue by cuiyanx** [01org/zephyr.js#899]
### Description

[W3C sensor.timestamp](https://www.w3.org/TR/generic-sensor/#sensor-timestamp):
```
8.1.3. Sensor.timestamp
The getter of the timestamp attribute returns latest reading["timestamp"]
```
Please add it for zephyr.js sensor.

### Test Code

Add console.log in [/samples/AmbientLight.js](https://github.com/01org/zephyr.js/blob/master/samples/AmbientLight.js):

`console.log("sensor.timestamp: " + sensor.timestamp);`

![selection_004](https://cloud.githubusercontent.com/assets/22387607/24340649/7038dcac-12e6-11e7-86ca-87a3f5844017.png)

 

### Actual Result

![selection_005](https://cloud.githubusercontent.com/assets/22387607/24342132/7489bb72-12f1-11e7-8c86-4a8a98ac4979.png)

 

### Expected Result

Get the latest reading value.
 

### Test Builds

Branch | Commit Id | Target Device | Test Date | Result
---|---|---|---|---
master | bb11854 | Arduino 101 | Mar 27, 2017 | Fail

**Comment by grgustaf** [comment]
Fixed in #942

**Comment by cuiyanx** [comment]
Thanks, be closed
